docs_nnx/guides/gemma.ipynb

```diff
@@ -23,20 +29,20 @@
    "outputs": [],
    "source": [
     "! pip install --no-deps -U flax\n",
-    "! pip install jaxtyping kagglehub treescope"
+    "! pip install jaxtyping kagglehub penzai"
```

Comment:
Why penzai? Doesn't treescope work?

Reply:
Correct. Penzai was removed in https://github.com/google/flax/commit/3895d51b008ee5440ade77c1cee72486710719f0.